target/linux/pistachio/marduk/target.mk

```diff
@@ -28,6 +28,7 @@ BOARDNAME:=marduk
 DEFAULT_PACKAGES+=kmod-leds-gpio kmod-ledtrig-heartbeat kmod-i2c-core i2c-tools \
                   kmod-sound-pistachio-soc alsa-lib alsa-utils alsa-utils-tests \
                   kmod-uccp420wlan kmod-cfg80211 fping iw hostapd wpa-cli wpa-supplicant \
+                  kmod-tpm-i2c-infineon kmod-random-tpm \
```

Comment:
Because gnutls package (which we have enabled) currently breaks the build as soon as it notices that trousers is available due to a missing reference 'tpmtool.c:(.text.startup+0x60c): undefined reference to pkcs11_common'.

Need to push fixes for that first. (Or we could enable GNUTLS_PKCS11 as a workaround for the bug)